### PR TITLE
feat: support extra http headers in browserType.connect()

### PR DESCRIPTION
**Pull request by yury-s**
https://github.com/microsoft/playwright-java/issues/398

### CODE REVIEW

File: docs/src/api/class-browsertype.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ This methods attaches Playwright to an existing browser instance.
 * langs: js
 - `params` <[Object]>
   - `wsEndpoint` <[string]> A browser websocket endpoint to connect to.
+  - `extraHTTPHeaders` <[Object]<[string], [string]>> Additional HTTP headers to be sent with web socket connect request. Optional.
```
Comment (pavelfeldman) [inline docs/src/api/class-browsertype.md:78]: Let's wait for Joel to remove `params` ?

Comment (pavelfeldman) [inline docs/src/api/class-browsertype.md:78]: these are not really 'extra', just `httpHeaders` or `headers`

**Review by pavelfeldman — APPROVED** [review]

File: docs/src/api/class-browsertype.md (hunk 1 above)
Comment (yury-s) [reply docs/src/api/class-browsertype.md:78]: sure

Comment (yury-s) [reply docs/src/api/class-browsertype.md:78]: done

Comment (yury-s) [reply docs/src/api/class-browsertype.md]: There are other headers as well but if think it's ok to user shorter name `headers`. Renamed.

**Review by aslushnikov — APPROVED** [review]

**Pull request merged**